lib/console/filter.rb

```diff
@@ -100,11 +100,11 @@ def verbose!(value = true)
 		end
 		
 		def off!
-			@level = -1
+			@level = self.class::MAXIMUM_LEVEL + 1
 		end
 		
 		def all!
-			@level = self.class::MAXIMUM_LEVEL
+			@level = -1
```

Comment:
Looking at this, I wonder what I was thinking when I wrote this. Seriously, it looks messed up.